docs/release-promotion/desktop/howto-rc.md

```diff
@@ -21,32 +21,10 @@ What's New Page setup should be done shortly after updates are available on the
 * Make the following changes to the `-No-WNP` version:
   * Append `-No-WNP` to the `name` field.
 * Make the following changes to the original file (eg: `Firefox-59.0-build1-No-WNP.json`:
```

Comment:
The original file is the `Firefox-59.0-build1.json` ain't it? The changes for the `Firefox-59.0-build1-No-WNP.json` have been done above already.

Also missing closing `)`